vms/avm/vm.go

```diff
@@ -56,6 +59,13 @@ var (
 	errGenesisAssetMustHaveState = errors.New("genesis asset must have non-empty state")
 
 	_ vertex.LinearizableVMWithEngine = (*VM)(nil)
+
+	// Metadata DB Prefixes
+	dbPrefixMetadata = []byte{0x0}
+	dbPrefixIndexer  = []byte{0x1}
+
+	// State DB Prefixes
+	dbPrefixState = []byte{0x1}
```

Comment:
I was thinking we'd have two prefixes, one for state and one for non-state. As an alternative, we could just have each db live under their own prefix under the root db instead of sharing a common metadata prefix. I don't see a reason to do/not do one over the other so open to suggestions here